src/DIRAC/ConfigurationSystem/Client/Helpers/CSGlobals.py

```diff
@@ -72,7 +25,14 @@ def getCSExtensions():
     Return list of extensions registered in the CS
     They do not include DIRAC
     """
-    return Extensions().getCSExtensions()
+    __csExt = []
```

Comment:
Can we get rid of `getCSExtensions` entirely? (and mark it as deprecated in rel-v7r3)

Reply:
It used:
- `DIRAC/Resources/Computing/SingularityComputingElement.py`
- `DIRAC/WorkloadManagementSystem/Agent/SiteDirector.py`